### PR TITLE
docs(engine/rest): set variables on process instance migration

### PR DESCRIPTION
**Pull request by tasso94**
related to CAM-13410

### CODE REVIEW

File: content/reference/rest/migration/execute-migration-async.md [hunk 1]
```diff
@@ -113,6 +113,14 @@ The migration plan JSON object has the following properties:
         </table>
     </td>
   </tr>
+  <tr>
+    <td>variables</td>
+    <td>
+      A map of variables which will be set into the process instances' scope.
+      Each key is a variable name and each value a JSON variable value object with the following properties:
+      {{< rest-var-request transient="false">}}
```
Comment (yanavasileva) [inline content/reference/rest/migration/execute-migration-async.md:121]: Transient should not be specified in order to work the check, at least for me locally the transient field was displayed with the above syntax:
```suggestion
      {{< rest-var-request>}}
```

**Review by yanavasileva — APPROVED** [review]
Looks good, please have a look at the two adjustments below before merging.

File: content/user-guide/process-engine/process-instance-migration.md [hunk 2]
```diff
@@ -236,6 +236,29 @@ MigrationPlan migrationPlan = processEngine.getRuntimeService()
...
+
+```java
+Map<String, Object> variables = Variables.putValue("my-variable", "my-value");
+MigrationPlan migrationPlan = processEngine.getRuntimeService()
+  .mapEqualActivities()
+  .setVariables(variables)
+  .build();
+```
+
+### Known Limitation
```
[the remark this answers is not in the document]
Comment (tasso94) [reply content/user-guide/process-engine/process-instance-migration.md:256]: 👍

File: content/reference/rest/migration/execute-migration-async.md (hunk 1 above)
Comment (tasso94) [reply content/reference/rest/migration/execute-migration-async.md:121]: 👍

**Review by yanavasileva — APPROVED** [review]

**Pull request merged**